startup_scripts/280_custom_links.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib.contenttypes.models import ContentType
+from extras.models import CustomLink
+from startup_script_utils import load_yaml
+import sys
+
+
+custom_links = load_yaml('/opt/netbox/initializers/custom_links.yml')
+
+if custom_links is None:
+  sys.exit()
+
+def get_content_type_id(content_type_str):
+  try:
+    return ContentType.objects.get(model=content_type_str).id
+  except ContentType.DoesNotExist:
+    print("⚠️ Error determining content type id for user declared var: {0}".format(content_type_str))
```

Comment:
```suggestion
    return None
```

... or ...

```suggestion
    pass
```

... (not sure which is more idiomatic in Python).

The error is technically accurate, but not actionable for our users.
They will not know what went wrong. And because we don't have access to the context here (i.e. the `custom_link.name` is not known, and it would be a drag to pass it to this function just in case there's an error), I would skip a message here and replace it with an error in the caller. (As per above suggestion.)